qcodes/station.py

```diff
@@ -684,3 +742,48 @@ def update_schema_file(
                 [qcodes.instrument_drivers] + additional_instrument_modules)
         ))
     )
+
+
+def _merge_yamls(*yamls: Union[str, Path]) -> IO[str]:
```

Comment:
didn't we discuss before that this IO output is unnecessary and a simple string output would work perfectly and make the usage of this function easier? :)
```suggestion
def _merge_yamls(*yamls: Union[str, Path]) -> str:
```

Reply:
Yes, we discussed it but according to my understanding it was not decided to definitely make it a string. Hence, I tried making it a str but got some errors so reverted it back. But you are right, I'll work on it and do the updates.